mlir/lib/Conversion/TosaToLinalg/TosaToLinalg.cpp

```diff
@@ -550,9 +549,20 @@ static Value expandRank(PatternRewriter &rewriter, Location loc, Value tensor,
   auto resultType =
       RankedTensorType::get(resultShape, shapedType.getElementType());
 
+  SmallVector<OpFoldResult> inputShape =
+      tensor::getMixedSizes(rewriter, loc, tensor);
+  SmallVector<OpFoldResult> outputShape;
+  if (failed(tensor::ExpandShapeOp::inferOutputShape(
```

Comment:
With the new build method you should be able to fold this into the `create` method? Here and everywhere else?

Reply:
Updated at all places, thanks!